stories/Atom/Reach-element/Table/table.scss

```diff
@@ -4,29 +4,123 @@
 
 /* table tag start */
 table {
-    border-spacing: 0px;
-  tr {
-    th {
-      vertical-align: top;
-      font-family: $Font-Family-ProximaRegular;
-      padding: 12px;
-      color: $color-black;
-      background-color: #edeff0;
-      text-align: left;
-      font-weight: 700;
-      @extend %paragragh-font-20;
-      @include devicebreak(medium) {
-        padding: 16px;
-      }
-    }
-    td {
-      vertical-align: top;
-      padding: 12px;
+  border-collapse: collapse;
+  border-spacing: 0px;
+  width: 100%;
+  @include devicebreak(medium) {
+    width: auto;
+  }
+  thead {
+    tr {
+      th {
+        vertical-align: top;
+        font-family: $Font-Family-ProximaSemiBold;
+        color: $color-black;
+        background-color: #edeff0;
```

Comment:
@harishkumar-srijan do we have a veriable for this color

Reply:
Done